client/src/state/history/readOneBlockQuery.js

```diff
@@ -7,36 +7,31 @@ import gql from 'graphql-tag';
 const query = gql`
 query ReadHistoryViewerBlock ($block_id: ID!, $limit: Int!, $offset: Int!) {
   readOneBlock(
-    Versioning: {
-      Mode: LATEST
+    versioning: {
+      mode: LATEST
     },
-    ID: $block_id
+    filter: { id: { eq: $block_id } }
   ) {
-    ID
-    Versions (limit: $limit, offset: $offset, sortBy: {
-      field: Version
-      direction: DESC
-    }) {
+    id
+    versions (limit: $limit, offset: $offset, sort: { version: DESC }) {
```

Comment:
ReadHistoryViewerBlock query fails with
{"data":null,"errors":[{"message":"Unknown argument \"limit\" on field \"versions\" of type \"Block\"."